test/kitchen-automate/recipes/configure_services.rb

```diff
@@ -0,0 +1,38 @@
+# Ensure the Automate Asset Store is enabled and trigger a reconfigure otherwise
+replace_or_add "Enable profiles in Automate" do
+  path "/etc/delivery/delivery.rb"
+  pattern "^compliance_profiles['enable'].*"
+  line "compliance_profiles['enable'] = true"
+  notifies :run, "execute[automate_reconfigure]", :immediately
+end
+
+execute 'automate_reconfigure' do
+  command <<-EOH
+    automate-ctl reconfigure
+  EOH
+  action :nothing
+end
+
+
+# Because we snapshot the instance, the initial S3 storage config goes away
+# Disable the S3 store for cookbooks:
+delete_lines "Avoid cookbook storage in S3" do
```

Comment:
we should add a note, that this is only suitable for test environments, not production environments